src/Views/IframeContentView.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Handle form template view skin.
+ *
+ * @package Give
+ */
+
+namespace Give\Views;
+
+/**
+ * Class IframeView
+ *
+ * @since 2.7.0
+ * @package Give
+ */
+class IframeContentView {
+	/**
+	 * Document page title.
+	 *
+	 * This will be use to setup title tag.
+	 *
+	 * @since 2.7.0
+	 * @var string
+	 */
+	protected $title;
+
+	/**
+	 * Document page body.
+	 *
+	 * This will be use to setup content of body tag.
+	 *
+	 * @since 2.7.0
+	 * @var string
+	 */
+	protected $body;
+
+	/**
+	 * Body classes.
+	 *
+	 * This will be use to setup body tag classes.
+	 *
+	 * @since 2.7.0
+	 * @var array
+	 */
+	protected $bodyClasses = [ 'give-form-templates' ];
+
+	/**
+	 * Set document page title.
+	 *
+	 * @param string $title
+	 *
+	 * @return IframeContentView $this
+	 */
+	public function setTitle( $title ) {
+		$this->title = $title;
+
+		return $this;
+	}
+
+	/**
+	 * Set document page title.
+	 *
+	 * @param string $body
+	 *
+	 * @return IframeContentView $this
+	 */
+	public function setBody( $body ) {
+		$this->body = $body;
+
+		return $this;
+	}
+
+	/**
+	 * Set document page title.
+	 *
+	 * @param array $classes
+	 *
+	 * @return IframeContentView $this
+	 */
+	public function setBodyClasses( $classes ) {
+		$this->bodyClasses = array_merge( $this->bodyClasses, (array) $classes );
+
+		return $this;
+	}
+
+
+	/**
+	 * Render view.
+	 *
+	 * Note: if you want to overwrite this function then do not forget to add action hook in footer and header.
+	 * We use these hooks to manipulated donation form related actions.
+	 *
+	 * @since 2.7.0
+	 */
+	public function render() {
+		ob_start();
```

Comment:
Any reason to use output buffering for this? I think it's more memory-intensive to do it this way, rather than just processing it like normal.

Reply:
And nothing we're doing in this function needs to be ran through output buffering, it's all normal HTML and normal WP function calls that have a getter that we can use